content/guides/merchant-integration-error-handling.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Merchant Integration Error Handling
+---
+
+Below are our guidelines for dealing with inconsistencies in [Payment Request](https://docs.centrapay.com/api/payment-requests#payment-request) statuses due to network issues or race conditions.
+
+## 🎯 Use the Payment Request Status as the Source of Truth
```

Comment:
@tom-blake do we want the emojis?
I think they look fine (see screenshot) but our other headings haven't had them 🤷‍♀️

Reply:
EDIT: I removed them as they looked bad in the table of contents.